client/package.json

```diff
@@ -37,7 +37,7 @@
     "graphql": "^0.7.0",
     "isomorphic-fetch": "^2.2.1",
     "react": "^15.2.1",
-    "react-ace": "^3.5.0",
+    "react-ace": "^4.0.0",
```

Comment:
Required to be able to override styles. See https://github.com/securingsincity/react-ace/commit/50e88e1417983c9349daefd2cd52fb44d9efc72c#diff-ac45432a2c51f02e94f43f5a1b4c8ef0